src/api/customAxiosInstance.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import hathorLib from '@hathor/wallet-lib';
+
+/**
+ * Create axios instance to be used in lib HTTP requests
+ *
+ * @module Axios
+ */
+
+/**
+ * Create an axios instance to be used when sending requests
+ *
+ * @param {callback} resolve Callback to be stored and used in case of a retry after a fail
+ * @param {number} timeout Timeout in milliseconds for the request
+ */
+const createRequestInstance = (resolve, timeout) => {
+  // Will override lib axios instance increasing the default request timeout
+  const instance = hathorLib.axios.defaultCreateRequestInstance(resolve, 30000);
```

Comment:
30 seconds? Even though it seems too much, we can keep it if the request processing is not cancelled when the request times out.

Reply:
As discussed privately, we will keep 30s for now until our endpoints have better performance.